relay-server/src/statsd.rs

```diff
@@ -48,6 +50,95 @@ impl GaugeMetric for RelayGauges {
     }
 }
 
+/// Gauge metrics collected from the Tokio Runtime.
+pub enum TokioGauges {
+    /// Exposes [`RuntimeMetrics::active_tasks_count`].
+    ActiveTasksCount,
+    /// Exposes [`RuntimeMetrics::blocking_queue_depth`].
+    BlockingQueueDepth,
+    /// Exposes [`RuntimeMetrics::budget_forced_yield_count`].
+    BudgetForcedYieldCount,
+    /// Exposes [`RuntimeMetrics::num_blocking_threads`].
+    NumBlockingThreads,
+    /// Exposes [`RuntimeMetrics::num_idle_blocking_threads`].
+    NumIdleBlockingThreads,
+    /// Exposes [`RuntimeMetrics::num_workers`].
+    NumWorkers,
+    /// Exposes [`RuntimeMetrics::worker_local_queue_depth`].
+    ///
+    /// This metric is tagged with:
+    /// - `worker`: the worker id
```

Comment:
nit (same with below):
```suggestion
    /// - `worker`: the worker id.
```